policyengine_us/tests/policy/baseline/gov/states/mi/tax/income/credits/home_heating/integration.yaml

```diff
@@ -0,0 +1,119 @@
+- name: integration test 1
+  period: 2022
+  input:
+    people:
+      person1:
+        is_full_time_student: false
+        claimed_as_dependent_on_another_return: false
+      person2:
+        is_full_time_student: true
+        claimed_as_dependent_on_another_return: false
+      person3:
+        is_full_time_student: true
+        claimed_as_dependent_on_another_return: false
+    tax_units:
+      tax_unit:
+        mi_household_resources: 1_000
+        mi_exemptions_count: 2
+        heating_expenses: 1_000
+        tax_unit_fpg: 23_030
+        utilities_included_in_rent: false
+        members: [person1, person2, person3]
+        tax_unit_size: 3
```

Comment:
this and `mi_exemptions_count` are inconsistent. they are also derived from the person-level information so neither is needed. in general for integration tests we should provide only "leaf-node" inputs - those without a formula. so also e.g. please remove tax_unit_fpg and mi_household_resources